pages/contributors.tsx

```diff
@@ -73,83 +76,286 @@ const ContributorsPage: FC<{ contributors: Contributor[] }> = ({
     (a, b) => b.contributions - a.contributions
   )
 
-  const buttonStyles =
-    'bg-violet-600 hover:bg-transparent text-white px-4 py-2 md:px-3 text-sm tracking-[.6px] rounded-md border border-dashed border-transparent duration-100 hover:border-violet-400 hover:text-violet-500 dark:hover:text-violet-400'
-
   const linkProps = {
     target: '_blank',
     rel: 'noopener noreferrer',
-    className: buttonStyles,
   }
 
-  const isDarkMode = resolvedTheme === 'dark'
+  const iconsComponents: { [key: string]: JSX.Element } = {
+    GitHub: <FaGithub />,
+    Twitter: <FaXTwitter />,
+    Sponsor: <FaHeart />,
+    LinkedIn: <FaLinkedin />,
+    Website: <FaStaylinked />,
```

Comment:
BTW, Why are we still using icons from the library? I think we should use our custom icons from Figma, later on in the future, we can plan to remove the icons library completely. The fewer the libraries are the faster the app will load.

What do you say?

Reply:
Sure, I would switch to the icons from the Figma file. Although, I prefer Icons from libraries unless a custom icon.

Reply:
I'm not making a custom component for Icons, otherwise we need to install tw-merge and then it will have some dependencies size.

Just using custom svg icon files for the same.

Reply:
Yeah, that's what I meant. Please, check the example in my sidebar PR.